drizztsaga/worldmap/bgee/bpwm_links_all.tbl

```diff
@@ -6,13 +6,11 @@ AR1000   E        F_7777     #WMtravel    6        1         N    N    N    N
 AR1000   E        F_0111     #WMtravel    4        1         N    N    N    N    N    0
 AR1000   E        F_0222     #WMtravel    4        1         N    N    N    N    N    0
 AR1000   E        F_9898     #WMtravel    6        1         N    N    N    N    N    0
-AR1000   E        F_7779     #Icewind03   6        1         N    N    N    N    N    0
```

Comment:
Maybe the proper fix should be to not reveal the area by the worldmap (only by a script). I will check  this before merge.

Reply:
This area is not creating in DLG or Script; it is only accessible via travel region or worldmap. This fix should be enough as it make it inaccessible from regular areas as it should has always been.
It would be safer to not reveal via worldmap but here both way works.